src/frontend/context/ObservationsContext.tsx

```diff
@@ -1,68 +1,51 @@
-// @flow
 import * as React from "react";
+import { Observation } from "mapeo-schema";
 
 import api from "../api";
-import type { LocationContextType } from "./LocationContext";
-import type { Status } from "../types";
+import { Status } from "../sharedTypes";
 
-export type ObservationAttachment = {
-  id: string,
-  type?: string,
-};
-
-export type ObservationValue = {
-  lat?: number | null,
-  lon?: number | null,
-  deviceId?: string,
-  metadata?: {
-    location?: LocationContextType | void,
-    manualLocation?: boolean,
-  },
-  refs?: Array<{ id: string }>,
-  attachments?: Array<ObservationAttachment>,
-  tags: { [string]: any },
-};
+export interface ObservationAttachment {
+  id: string;
+  type?: string;
+}
 
-export type Observation = {
-  id: string,
-  version: string,
-  created_at: string,
-  timestamp?: string,
-  userId?: string,
-  type: "observation",
-  links?: string[],
-  schemaVersion: 4,
-  value: ObservationValue,
-};
+export interface ClientGeneratedObservation
```

Comment:
Maybe not the best-named type but wanted it to be very clear that the this is for representing observations that are being created client-side i.e. without meta information that's added from the DB-side